packages/lit-html/src/lit-html.ts

```diff
@@ -558,6 +576,12 @@ class Template {
       nodeIndex++;
     }
   }
+
+  _createElement(html: string) {
```

Comment:
Note this is an override point for ShadyDOM?

Reply:
Added.